cloud_build/py27_docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:2.7
+
+RUN wget https://github.com/googleapis/google-cloud-python/archive/master.zip
```

Comment:
Maybe we can use a chained RUN in Dockerfile?

Reason: https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better
Example: https://github.com/GoogleCloudPlatform/cloud-sdk-docker/blob/master/Dockerfile

Reply:
Done.